frontend/src/components/explorer/InteractionPartners.vue

```diff
@@ -20,46 +20,31 @@
           </div>
         </div>
         <br><br>
```

Comment:
I think it looks better without the line break

```suggestion
```

Reply:
I agree, thanks for the suggestion!